### DISCUSSION TITLE
Where do I find the AH odds?

### DISCUSSION THREAD

**Discussion by vuminhquang** [cloudbet/docs#24] [General]
Hello,

I used this API to get odds of a match. But don't see the odd.
​/v2​/odds​/events​/{id}

![image](https://user-images.githubusercontent.com/20512463/162207744-c5c67bdf-b85b-4284-9ac9-8eb03655a7d2.png)
What is the field price here?
And the probability: when get 1/0.288 = 3.4. Odd I get from UI is 1.6 only. Please make clear for me. Thank you.

![image](https://user-images.githubusercontent.com/20512463/162207910-0e7eff4d-ad06-416f-9107-acc5a5254023.png)

**Comment by kgravenreuth** [comment]
Hi @vuminhquang ,

The `Price` is indeed the odds in our API.

For an explanation of the Asian handicap odds and why they appear so here, please refer to #26
